src/utils/calc-order-profit-loss-percents.js

```diff
@@ -0,0 +1,49 @@
+import BigNumber from 'bignumber.js';
+import { BUY } from 'modules/trade/constants/types';
+
+/**
+ *
+ * @param numShares number of shares the user wants to buy or sell
+ * @param limitPrice maximum price for purchases or minimum price for sales the user wants to purchase or sell at
+ * @param side BUY or SELL; whether or not the user wishes to buy or sell shares
+ * @param minValue only relevant for scalar markets; all other markets min is created and set to 0
+ * @param maxValue only relevant for scalar markets; all other markets max is created and set to 1
+ * @returns object with the following properties
+ *    potentialEthLoss:       number, maximum number of ether that can be lost according to the current numShares and limit price
+ *    potentialEthProfit:     number, maximum number of ether that can be made according to the current numShares and limit price
+ *    potentialProfitPercent: number, the maximum percentage profit that can be earned with current numShares and limit price,
+ *                                    excluding first 100% (so a 2x is a 100% return and not a 200% return). For BUYs, loss is always 100% (exc. fees)
+ *    potentialLossPercent:   number, the max percentage loss that can be lost with current numShares and limit price; for SELLs loss is always 100%
+ */
+
+export default function (numShares, limitPrice, side, minValue, maxValue, type) {
```

Comment:
Great suggestion!  While I do generally prefer an options object, for consistency in the UI CB, let’s leave the multiple params in place here for now.